src/executables/util_sa.c

```diff
@@ -554,7 +555,7 @@ createdb (UTIL_FUNCTION_ARG * arg)
 	}
       fprintf (stderr, msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_CREATEDB, CREATEDB_MSG_FAILURE));
       PRINT_AND_LOG_ERR_MSG (msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_CREATEDB, CREATEDB_MSG_BAD_RANGE),
-			     prm_get_name (PRM_ID_LOG_VOLUME_SIZE), vol_buf, min_buf, max_buf);
+			     prm_get_name (PRM_ID_LOG_VOLUME_SIZE), vol_buf, max_buf);
```

Comment:
Since we don't round up the number of log pages, the message would be 
```
The given 'log_volume_size', 10m is out of range.  Maximum supported size is 4.0G.
```

It was
```
The given 'log_volume_size', 10m is out of range. Choose a size between 20.0M and 4.0G.
```

I'd like to propose to keep the messages and just modify round up the number of db pages.